src/documentation/TableWithFilter/ReactSelectWrapper.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Select, { components } from "react-select";
+import Input from "../../components/Input";
+import classnames from "classnames";
+import { useSelector } from "react-redux";
+
+export default function ReactSelectWrapper(props) {
+  const { className, control, onChange, labelText, value } = props;
```

Comment:
Some of these props are not used later on.

Reply:
ooohh okay